### DISCUSSION TITLE
Bulk Insert of Rows from a CSV or TSV or Excel Copy.

### DISCUSSION THREAD

**Discussion by sjugraj** [directus/directus#9342] [Feature Requests]
Is there a way to bulk insert rows via the UI.

like if there is a copy in clipboard from a tab separated values or others. We can create place holder rows to fill and correct any misaligned cells then trigger a insert post request as whole once all the conflicts resolved.

**Comment by rijkvanzanten** [comment]
Not from the admin app yet

**Reply by sjugraj** [reply]
is there any feature request i can track or can we create it?

**Reply by benhaynes** [reply]
Quite a few... https://github.com/directus/directus/discussions?discussions_q=csv

I think this has the most traction: https://github.com/directus/directus/discussions/7246

**Comment by azrikahar** [comment]
Implemented in #11050

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
